### PR TITLE
test ruby 2.7

### PR DESCRIPTION
**Pull request by seuros**
removing warnings and dead code too.

### CODE REVIEW

File: lib/trailblazer/activity.rb [hunk 1]
```diff
@@ -7,11 +7,9 @@ def initialize(schema)
       @schema = schema
     end
 
-    def call(args, circuit_options={})
-      @schema[:circuit].(
-        args,
-        circuit_options.merge(activity: self)
```
[the remark this answers is not in the document]
Comment (seuros) [reply lib/trailblazer/activity.rb:13]: I will have to do `**(circuit_options.merge(activity: self))` or use `opts`

File: lib/trailblazer/activity/structures.rb [hunk 2]
```diff
@@ -18,24 +12,24 @@ def initialize(semantic:, **options)
         @options = options.merge(semantic: semantic)
       end
 
-      def call(args, circuit_options)
-        return self, args, circuit_options
```
[the remark this answers is not in the document]
Comment (seuros) [reply lib/trailblazer/activity/structures.rb:22]: This is is rubocop and static code analyser.

File: lib/trailblazer/activity.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (apotonick) [reply lib/trailblazer/activity.rb:13]: the first one looks ok to me

[the remark this answers is not in the document]
Comment (seuros) [reply lib/trailblazer/activity.rb:13]: Done

**Comment by seuros** [comment]
Finally :)

**Pull request merged**